chcemvediet/apps/inforequests/urls.py

```diff
@@ -28,20 +29,22 @@
     u'appeal':                    lazy_concat(_(u'inforequests:urls:appeal'), u'/'),
     u'snooze':                    lazy_concat(_(u'inforequests:urls:snooze'), u'/'),
     u'attachments':               lazy_concat(_(u'inforequests:urls:attachments'), u'/'),
+    u'attachment_finalizations':  lazy_concat(_(u'inforequests:urls:attachment_finalizations'), u'/'),
```

Comment:
Toto zobrazi `attachment_finalizations` na url: `/infoziadosti/inforequests:urls:attachment_finalizations/4/`

Kdezto `attachments` sa zobrazuju na url: `/infoziadosti/prilohy/18/`

Ked si si to skusal, neprisla Ti ta url divna? Skus samostatne zistit, kde sa nastavuje url pre `inforequests:urls:attachments` a daj tam normalnu url aj namiesto `inforequests:urls:attachment_finalizations`.

Idealne nieco ako `/infoziadosti/prilohy/anonymizovane/4/` namiesto `/infoziadosti/inforequests:urls:attachment_finalizations/4/`

Reply:
:ok: